### DISCUSSION TITLE
why bind not work in customtk

### DISCUSSION THREAD

**Discussion by Zeidev** [TomSchimansky/CustomTkinter#806] [Q&A]
```
def button_function():
    print("button pressed")

button = customtkinter.CTkButton(master=app, text="CTkButton", command=button_function)
button.bind('<Return>',button_function)
button.place(relx=0.5, rely=0.5, anchor=tkinter.CENTER)
```

this code working on tkinter but not work customtkinter

**Comment by TomSchimansky** [comment]
It does not work because the button is not focused with `button.focus()`. But if you want to catch the return let event I would suggest to bind to the main window instead.

**Reply by Zeidev** [reply]
can you give any example ?
